cms/static/sass/style-app.scss

```diff
@@ -1,46 +1,51 @@
 // studio - css architecture
 // ====================
 
-// Table of Contents 
+// Table of Contents
 // * +Libs and Resets
-// * +Vendor and Rebase 
+// * +Vendor and Rebase
 // * +Base - Utilities
 // * +Base - Assets
 // * +Base - Starter
 // * +Base - Elements
+// * +Base - Specific Views
+// * +Base - Contexts
+// * +Xmodule
 
 // +Libs and Resets - *do not edit*
-// ==================== 
+// ====================
 @import 'bourbon/bourbon'; // lib - bourbon
 @import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
 
 // +Vendor and Rebase - *referenced/used vendor presentation and reset*
-// ==================== 
+// ====================
 @import 'reset';
 
 // +Base - Utilities
-// ==================== 
+// ====================
 @import 'variables';
 @import 'mixins';
 @import 'mixins-inherited';
 
 // +Base - Assets
-// ==================== 
+// ====================
 @import 'assets/fonts';
 @import 'assets/graphics'; // sprites, basic image/figure/svg styling
 @import 'assets/anims'; // animations
 
 // +Base - Starter
-// ==================== 
+// ====================
 @import 'base';
 
 // +Base - Elements
-// ==================== 
+// ====================
 @import 'elements/typography';
 @import 'elements/icons'; // references to icons used
+@import 'elements/navigation'; // all archetypes of navigation
 @import 'elements/controls'; // buttons, link styles, sliders, etc.
+@import 'elements/pagination'; // pagination
+@import 'elements/xblocks'; //  studio rendering chrome for xblocks
 @import 'elements/modules'; //  content module patterns
-@import 'elements/navigation'; // all archetypes of navigation
```

Comment:
Do you want to introduce reordering in this PR? Seems like a better follow-up to reduce risk unless there is a good reason for now...